articles/kubernetes-fleet/resource-propagation.md

```diff
@@ -58,7 +58,7 @@ spec:
     placementType: PickAll
     affinity:
         clusterAffinity:
-            requiredDuringSchedulingIgnoredDuringExection:
+            requiredDuringSchedulingIgnoredDuringExecution:
```

Comment:
This has been fixed already in the current version. Could you abandon the PR?

Reply:
Yes, thanks for the update.